app/main.js

```diff
@@ -13,7 +13,7 @@ const addMenu = require('./menu').addMenu;
 const winston = require('winston')
 
 let backendProcess = undefined
-let powerSaver = undefined
+let powerSaver_ID = undefined
```

Comment:
try to avoid underscores in JS var names as that is more common for JS vars. ie `powerSaverID`